legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/GitLabServerHealthCheck.java

```diff
@@ -15,26 +15,23 @@
 package org.finos.legend.sdlc.server.gitlab;
 
 import com.codahale.metrics.health.HealthCheck;
-import org.finos.legend.sdlc.server.gitlab.mode.GitLabModeInfos;
 
 public class GitLabServerHealthCheck extends HealthCheck
 {
-    private final GitLabModeInfos gitLabModeInfos;
-
-    private GitLabServerHealthCheck(GitLabModeInfos gitLabModeInfos)
+    private GitLabServerHealthCheck(GitLabConfiguration config)
     {
-        this.gitLabModeInfos = gitLabModeInfos;
+        // TODO depending how we plan to do health check, use the supplied config accordingly
     }
 
     @Override
     protected Result check()
     {
         // TODO do a more substantive health check
-        return this.gitLabModeInfos.isEmpty() ? Result.unhealthy("No GitLab modes available") : Result.healthy();
+        return Result.healthy();
```

Comment:
At the very least, let's maintain the current level of validation by checking that server and app info have been configured. If possible, we should check that the GitLab server is accessible.

Reply:
I will only check for the app info for now, will leave a comment regarding your suggestion